docs/data-cookbooks/setup.md

```diff
@@ -74,37 +70,33 @@ Rules are used by to start processing workflows and the transformation process.
 
 The Cumulus Core repository has an example of a Kinesis rule [here](https://github.com/nasa/cumulus/blob/master/example/data/rules/L2_HR_PIXC_kinesisRule.json).
 
-|Key|Value|Required|Description|
-|---|-----|--------|-----------|
+|Key  |Value  |Required|Description|
+|:---:|:-----:|:------:|-----------|
 |name|`"L2_HR_PIXC_kinesisRule"`|Yes|Name of the rule. This is the name under which the rule will be listed on the dashboard|
 |workflow|`"KinesisTriggerTest"`|Yes|Name of the workflow to be run. A list of available workflows can be found on the Workflows page|
-|provider|`"PODAAC_SWOT"`|Yes|Configured provider's ID. This can be found on the Providerse page|
-|collection|`<JSON Object>` collection object shown [below](#collection-object)|Yes|Name and version of the collection this rule will moderate. Configured and found in the Collections page|
-|rule|`<JSON Object>` rule type and associated values - discussed [below](#rule-object)
-|state|`ENABLED`|Yes|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active|
+|provider|`"PODAAC_SWOT"`|No|Configured provider's ID. This can be found on the Providers dashboard page|
+|collection|`<JSON Object>` collection object shown [below](#collection-object)|Yes|Name and version of the collection this rule will moderate. Relates to a collection configured and found in the Collections page|
+|rule|`<JSON Object>` rule type and associated values - discussed [below](#rule-object)|Yes|Object defining the type and subsequent attributes of the rule|
+|state|`ENABLED`|No|<code>(ENABLED&#124;DISABLED)</code> whether or not the rule will be active (defaults to `ENABLED`)|
```

Comment:
so this is still not true yet?  I didn't hear from Lauren if this was going to get fixed or not?